_layouts/page.html

```diff
@@ -15,6 +15,11 @@
   {% if page.author.gplus %}<link rel="author" href="https://plus.google.com/{{ page.author.gplus }}">{% endif %}
   <meta property="og:title" content="{% if page.title %}{{ page.title }} {% else %}{{ site.inc.title }}{% endif %}"/>
   <meta property="og:url" content="{{ site.url }}{{ page.url }}"/>
+  {% if page.image %}
+  <meta content="/images/site-logo.svg" property="og:image">
```

Comment:
Seems my comment via email was lost, here it is:

Oh ok. Having a screenshot like in Nicola's post breaks the preview anyway, so I'm fine with just showing the upday logo. In this case I'd remove the conditional though and just put the meta tag.